templates/contribute/index.md

```diff
@@ -51,7 +51,7 @@ Typical workflow:
 * Sometimes you may not want to create a new branch, but instead work on a branch
   that someone else created, or you created from a different computer.
   In that case you need to use `git switch their_new_branch` (note there is no `-c` here).
-* Make local changes, e.g. using Visual Studio Code using the Lean extension.
+* Make local changes, e.g. using Visual Studio Code using the Lean extension. If you created new files, use `scripts/mk_all.sh`.
```

Comment:
Can you make this a separate bullet point, after `lake build`?

Reply:
Sure!